src/azure-cli/setup.py

```diff
@@ -76,7 +76,7 @@
     'azure-mgmt-containerinstance~=1.4',
     'azure-mgmt-containerregistry==3.0.0rc16',
     'azure-mgmt-cosmosdb~=3.0.0',
-    'azure-mgmt-containerservice~=9.4.0',
+    'azure-mgmt-containerservice~=11.1.0',
```

Comment:
pls rerun live test and re-recording tests, to avoid regression and update tests.

Reply:
test recording updated.